core/http/src/parse/indexed.rs

```diff
@@ -31,7 +31,7 @@ impl AsPtr for [u8] {
 }
 
 
-#[derive(PartialEq)]
+#[derive(PartialEq, Clone)]
```

Comment:
This is incorrect as it adds a `Clone` bound to `T`, which is unnecessary. `Indexed` already implements `Clone`; no changes are needed to `indexed.rs`.